docs/integrations/digitalocean/gslb.mdx

```diff
@@ -0,0 +1,315 @@
+
+Your demo API is already globally load-balanced, but if you want to extend your ngrok usage even further, you can enable one or more [Traffic Policy modules](/docs/http/traffic-policy/index.mdx) on your Edge to control traffic or establish standards for how your upstream services are accessed.
+
+You have two options:
+
+- Using `curl` with the ngrok API's [`/edges/https/{edge_id}/routes/{id}/policy` endpoint](/docs/api/resources/edge-route-policy-module.mdx). For this step, you will need to have exported the `ROUTE_ID` environment variable in step 4
+
+  ```bash
+  curl \
+    -X PUT "https://api.ngrok.com/edges/https/${EDGE_ID}/routes/${ROUTE_ID}/policy" \
+    -H "Authorization: Bearer ${NGROK_API_KEY}" \
+    -H "Content-Type: application/json" \
+    -H "Ngrok-Version: 2" \
+    -d '{"enabled":true,"outbound":[{"actions":[{"type":"add-headers","config":{"headers": {"is-ngrok": "1","country": "${.ngrok.geo.country_code}"}}}],"name":"Add ngrok headers"}]}'
+  ```
+
+- In the Traffic Policy module section of your [Edge](https://dashboard.ngrok.com/cloud-edge/edges) in the ngrok dashboard, where you can click the **Edit Traffic Policy** button and then the **YAML** button and add the YAML below.
+
+  ```yaml
+  inbound: []
+  outbound:
+    - expressions: []
+      name: Add ngrok headers
+      actions:
+        - type: add-headers
+          config:
+            headers:
+              country: ${.ngrok.geo.country_code}
+              is-ngrok: "1"
+  ```
+
+When you request your API again, the response includes the `country` (with the appropriate code for where you're generating said request) and `is-ngrok` headers.
+
+```bash
+curl -s -I -X GET https://<YOUR_NGROK_DOMAIN>/api
+
+HTTP/2 200
+content-type: application/json
+country: US
+date: Wed, 22 May 2024 14:53:42 GMT
+is-ngrok: 1
+content-length: 63
+```
+
+## What's next?
+
+You now have a globally load-balanced API deployment using three DigitalOcean VMs, three ngrok agents, three secure tunnels, one ngrok Edge, and a single convenient endpoint for your users.
+
+From here, you have many options for extending your use of ngrok's GSLB:
+
+- Spread the load from user requests further by creating additional deployments in more regions, adding a new Tunnel Group backend for each, and [patching](https://ngrok.com/docs/api/resources/weighted-backends/#update-weighted-backend) your Weighted Backend configuration.
+- Add more VMs to a region with existing deployments and add them to the relevant Tunnel Group backend. Ngrok will then load-balance between those specific VMs equally _after_ weighting requests on the Backend level.
+  - Alternatively, use [NodeBalancers](https://www.DigitalOcean.com/docs/products/networking/nodebalancers/) for
+- Provision a Kubernetes cluster with the same workload and the ngrok [Kubernetes Operator](/docs/k8s/index.mdx) to load-balance between VM- and Kubernetes-based deployments of the same API or application.
```

Comment:
Provision a Kubernetes cluster with the same workload and the ngrok [Kubernetes Operator](/docs/k8s/index.mdx) to load balance between VM- and Kubernetes-based deployments of the same API or application.